docs/download.md

```diff
@@ -4,44 +4,60 @@ title: Download
 sidebar: nav.html
 ---
 
-{% include anchor.html class="h3" title="File Download" hash="file" %}
+{% include anchor.html class="h3" title="Download" hash="file" %}
 
-The PouchDB file should come before any files that use it.
-
-For use in an HTML file:
+Quick start:
 
 {% highlight html %}
-<script src="https://cdn.jsdelivr.net/pouchdb/{{ site.version }}/pouchdb.min.js"></script>
+<script src="https://cdn.jsdelivr.net/pouchdb/{{ site.version }}/pouchdb.min.js">
+</script>
```

Comment:
Adding a linebreak prevents the word `script` from getting broken up. I wish we had a shorter CDN link.